fasthtml_hf/deploy.py

```diff
@@ -51,7 +51,7 @@ def deploy(
     token:str=None, # Hugging Face token for authentication
     python_ver:str='3.10', # Version of python to use
     upload:bool_arg=True, # Set to `false` to skip uploading files
-    private:bool_arg=False): # Make the repository private
+    private:lambda x: bool(bool_arg(x))=False): # Make the repository private
```

Comment:
This is just a _placeholder_ suggestion. 

It might be better to update fastcore to include a version of `bool_arg` which casts the string to True/False rather than 0/1.  
Or perhaps we could just cast private to a bool in the deploy method.

Reply:
How about just adding a line inside this method that casts it?